wum-live-sync-automation/uatBatchJob/Jenkinsfile

```diff
@@ -0,0 +1,176 @@
+        if(childJobName != null) {
+            
+            if(!addUpdateDBDetailsOnly) {
+                updateListJson[updateListItemNo] = jsonSlurper.parseText("{}")
+                updateListJson[updateListItemNo].productName = row.product_name
+                updateListJson[updateListItemNo].productVersion = row.product_version
+                updateListJson[updateListItemNo].jobName = childJobName
+                
+                updateListJson[updateListItemNo].updateFileName = updateFileName
+                updateListJson[updateListItemNo].updateFileUrl = env.JOB_URL + "ws/" + wsUpdateFolderName + "/" + updateFileName
+                
+                updateListJson[updateListItemNo].updateDBDetails = jsonSlurper.parseText("[]")
+            }
+            
+            updateListJson[updateListItemNo].updateDBDetails[updateDBDetailsItemNo] = jsonSlurper.parseText("{}")
+            updateListJson[updateListItemNo].updateDBDetails[updateDBDetailsItemNo].id = row.id
+            updateListJson[updateListItemNo].updateDBDetails[updateDBDetailsItemNo].updateNo = row.update_no
+            updateListJson[updateListItemNo].updateDBDetails[updateDBDetailsItemNo].timestamp = row.timestamp
+        } else {
+            
+            println "WARN: There is no " + childJobNameFormat + " exist. Please create a WUM child job for this."
+        }
+        
+    } else {
+        
+        println "WARN: For " + row.product_name + "-" + row.product_version + "no WUM updated pack exist, thus removing rowId: [" +  row.id + "], updateNo: [" + row.update_no + "] from UAT testing."
+    }
+}
+
+def dbConnection = null
+
+try{
+    
+    // Date manipulation for query string
+    def dateFromat = "YYYY-MM-dd"
+    def now = new Date()
+    def todayDate = now.format(dateFromat)
+    def yesterdayDate = now.minus(1).format(dateFromat)
+    
+    // Get environment variables
+    def build = this.getProperty('binding').getVariable('build')
+    def listener = this.getProperty('binding').getVariable('listener')
+    def env = build.getEnvironment(listener)
+    
+    def wsUpdateFolderName = "wumUpdates"
+    
+    // Creating WUM_release view job names list
+    def wumReleaseJobNamesList = []
+    hudson.model.Hudson.instance.getView('WUM_Release').items.each() {
+        wumReleaseJobNamesList.add(it.fullDisplayName)
+    }
+    
+    println "=============Running Groovy shell commands start==============="
+    
+    // find . -name \wso2*-[0-9].[0-9].[0-9]?*.zip -type f -delete - find wso2 timestamp updates in products and delete recursively
+    def wumUpdateFileNameList = []
+    def wumProdBaseLoc = "/home/ubuntu/.wum-wso2/products/"
+    def wumPackLoc = "/build/jenkins-home/workspace/wumInstall/wum"
+    def shellRunStatus = runShellCommands(env, wumProdBaseLoc, 
+        "mkdir -p " + env.WORKSPACE + "/" + wsUpdateFolderName + "/ && " +
+        "find " + wumProdBaseLoc + " -name \\wso2*-[0-9].[0-9].[0-9]?*.zip -type f -delete && " +
```

Comment:
This regex does match the API-M's version _1.10.0_. Btw, we need to escape the dot char.